### PR TITLE
Feature/student unable view add cohort

### PR DESCRIPTION
**Pull request by dan1212111**
Added a feature to hide the 'addCohort` button.

-works with the correct navigation/but not if manually type the `url` route.

### CODE REVIEW

File: src/components/Header/Header.jsx
Comment (Auenc) [inline src/components/Header/Header.jsx:76]: Does this condition need to be here now?

If you're setting `display: none` if the role is a student, do we need to conditionally render the button at all?

It seems the useEffect and this conditional render are at odds with eachother.

**Review by Auenc — CHANGES_REQUESTED** [review]
There are a lot of formatting changes here. So it might be a good idea to wait until the poll is over before we decide to merge, just so we can follow the results of that poll. 

The biggest problem to address here is the question around the conditional render. I think either we keep the conditional render and remove the useEffect, or remove the conditional render and keep the useEffect.

**Comment by Auenc** [comment]
As per the result of the poll, the cohort has decided to use single quotes. Could you please change your work to reflect this :)

File: src/components/Header/Header.jsx
Comment (dan1212111) [reply src/components/Header/Header.jsx]: I've taken out the conditional rendering.

**Review by Auenc — CHANGES_REQUESTED** [review]
The code looks good, but you're using the wrong quote style. Please update this PR so it uses the correct style and has semi-colons at the end of each line.

**Review by Auenc — DISMISSED** [review]
LGTM

**Review by KinTale — DISMISSED** [review]
nice refactor

**Comment by marianphillips** [comment]
I think with the new way the header has been fixed, this bug has now been fixed?

**Pull request closed without merging**
